### ISSUE TITLE
[FEATURE] `wash label` should support multiple label key-value pairs

### ISSUE THREAD

**Issue by connorsmith256** [wasmCloud/wasmCloud#1286]
### Affected project(s)

- [ ] documentation
- [ ] examples
- [ ] wasmCloud host
- [X] wasmCloud CLI (wash)
- [ ] wasmCloud dashboard UI (washboard)
- [ ] capability providers
- [ ] provider bindgen
- [ ] control interface client
- [ ] other / not sure

### Is your feature request related to a problem? Please describe.

Today, if I want to add multiple labels to a host, I have to submit multiple commands:
```
wash label my-host foo=bar
wash label my-host baz=qux
```

### Describe the solution you'd like

Ideally, I could simply `wash label my-host foo=bar baz=qux`

### Additional context

Note the control interface/host currently accept one label at a time, so wash would be responsible for submitting multiple commands. Alternatively, the host + control interface could support receiving a batch of labels at once.

**Comment by RohanMishra315** [comment]
Hey @connorsmith256 Let me look into this ?

**Comment by connorsmith256** [comment]
@RohanMishra315 sounds great! Let us know if you have any questions

**Comment by hanshal101** [comment]
Hey @RohanMishra315 are you still working on this issue?

**Comment by thomastaylor312** [comment]
@hanshal101 It has been a month with no PR, so I think you're ok to take this if you'd like!

**Comment by hanshal101** [comment]
Ok will work on this issue. I'll take some time as I'm new to rust and WASMCLOUD

**Comment by hanshal101** [comment]
Hey, @connorsmith256 and @thomastaylor312 Please correct me if I'm wrong here. We need to add multiple label options at once.
For this we need to modify the below code. https://github.com/connorsmith256/wasmCloud/blob/08d1171f1c8e2e3d1fd84740a575b6fb73056aa2/crates/wash-lib/src/cli/label.rs#L61-L79
```
else {
        if value.is_empty() {
            bail!("No value provided");
        }

        let ack = client
            .put_label(&host_id, key, value)
            .await
            .map_err(boxed_err_to_anyhow)?;
        if !ack.accepted {
            bail!("Operation failed: {}", ack.error);
        }

        Ok(CommandOutput::from_key_and_text(
            "result",
            format!("Host `{}` labeled with `{}={}`", friendly_name, key, value),
        ))
    }
```
The expected code to enable this feature, according to me, would be:

```
else {
        let mut labels_map = HashMap::new();
        for label in &cmd.labels {
            let (key, value) = label.split_once('=').unwrap_or((label.as_str(), ""));
            if value.is_empty() {
                bail!("No value provided for label '{}'", key);
            }
            labels_map.insert(key.to_string(), value.to_string());
        }

        for (key, value) in labels_map.iter() {
            let ack = client
                .put_label(&host_id, key, value)
                .await
                .map_err(boxed_err_to_anyhow)?;
            if !ack.accepted {
                bail!("Operation failed: {}", ack.error);
            }
        }
}
```

Please do review the following changes and guide me.
Thank You

**Comment by thomastaylor312** [comment]
@hanshal101 Yep, that looks like you have the right idea! You can probably simplify your example code by doing something like 

```
cmd.labels.into_iter().map(|s| { splitting code here })
```

And then you can do the `for` loop for putting the label

**Comment by hanshal101** [comment]
Ok are we looking for this:
```
else {
  let labels_map: HashMap<_, _> = cmd.labels.into_iter()
      .map(|label| {
          let (key, value) = label.split_once('=').unwrap_or((label.as_str(), ""));
          if value.is_empty() {
              bail!("No value provided for label '{}'", key);
          }
          (key.to_string(), value.to_string())
      })
      .collect::<Result<_, _>>()?;
  
  for (key, value) in labels_map.iter() {
      let ack = client
          .put_label(&host_id, key, value)
          .await
          .map_err(boxed_err_to_anyhow)?;
      if !ack.accepted {
          bail!("Operation failed: {}", ack.error);
      }
  }

  Ok(CommandOutput::from_key_and_text(
      "result",
      format!("Host `{}` labeled with provided labels", friendly_name),
  ))
}
```

**Comment by hanshal101** [comment]
Hey @thomastaylor312 can you review this.

**Comment by thomastaylor312** [comment]
@hanshal101 Please go ahead and [open a PR](https://github.com/wasmCloud/wasmCloud/compare) with your changes and we will review it there

**Comment by vados-cosmonic** [comment]
Solved by #2140
